uid2_client/refresh_keys_util.py

```diff
@@ -0,0 +1,68 @@
+import datetime as dt
+import json
+from datetime import timezone
+
+from .keys import EncryptionKey, EncryptionKeysCollection
+from .request_response_util import *
+from .identity_scope import IdentityScope
+
+
+def _make_dt(timestamp):
+    return dt.datetime.fromtimestamp(timestamp, tz=timezone.utc)
+
+
+def parse_keys_json(resp_body):
+    keys = []
+    identity_scope = IdentityScope.UID2
+    if resp_body["identity_scope"] == "EUID":
+        identity_scope = IdentityScope.EUID
+    for key in resp_body["keys"]:
+        keyset_id = key.get("keyset_id")
+        key = EncryptionKey(key['id'],
+                            key.get('site_id', -1),
+                            _make_dt(key['created']),
```

Comment:
Would you like to replace all method to get(), like key.get("created")?

Reply:
for response fields which are necessary for the key to work, it makes sense to alert the user in case of a missing value. So `key['created']` will throw an exception if the field is absent. 
Using `key.get("created")` it will silently fail. I am assuming that alerting the user by throwing an exception is the right way to go